server/public/model/preference.go

```diff
@@ -60,6 +60,8 @@ const (
 	PreferenceEmailIntervalHour              = "hour"
 	PreferenceEmailIntervalHourAsSeconds     = "3600"
 	PreferenceCloudUserEphemeralInfo         = "cloud_user_ephemeral_info"
+
+	MaxPreferenceValueLength = 20000
```

Comment:
I wanted to still keep an arbitrary limit to prevent extreme edge cases. If we find that this somehow isn't enough, we can always keep increasing it up to the 64k limit that MySQL has, but this is more than enough to fix the issue reported in the ticket.